testdata/013_console_assert.ts.out

```diff
@@ -0,0 +1 @@
+You will see me.

```

Comment:
it will be: "Assertion failed: You will see me. "

Reply:
you can do:
./deno 013_console_assert.ts > 013_console_assert.ts.out

Reply:
Sorry, that is my fault. I will update.

Reply:
@g-plane No problem : )

Reply:
you should put one blank space ` ` at the end of your `.out` file.
see: stringifyArgs:
https://github.com/ry/deno/blob/7b5fbc71970056515c41f6983fc8e1f8ee443d11/globals.ts#L39-L49